recipes/botorch/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "botorch" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/botorch/archive/v{{ version }}.tar.gz
```

Comment:
We could use the tarballs on pypi, but the license file isn't currently distributed with the source. I've put in a PR so that this might happen in future releases:

https://github.com/pytorch/botorch/pull/499

If you switch, here are directions for manually packaging the license file:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Reply:
Seems that the PR has been merged. 
I propose switching to pypi starting release > 0.3.0 ?